### PR TITLE
Add Copy Benchmark

### PR DESCRIPTION
**Pull request by Arleee1**

### CODE REVIEW

File: misc-bench/cpp-copy/baselines/CPU/copy.cpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#include <getopt.h>
+#include <assert.h>
+#include <stdint.h>
+#include <iomanip>
+#include <chrono>
+#include <omp.h>
+#include <iostream>
+#include <vector>
+
+#include "utilBaselines.h"
```
Comment (fasiddique) [inline misc-bench/cpp-copy/baselines/CPU/copy.cpp:19]: The util file is in the misc-bench directory. So I assume this include will not work?

Comment (Arleee1) [reply misc-bench/cpp-copy/baselines/CPU/copy.cpp:19]: Yes, as I have included the directory in the include path in the Makefile. I can reformat it to be like the others if you would like though.

File: misc-bench/cpp-copy/baselines/GPU/copy.cu [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    gpu_y[idx] = gpu_x[idx];
+  }
+}
+
+/**
+ * @brief Main of the Host Application.
+ */
+int main(int argc, char **argv)
+{
+  cudaGetLastError();
```
[the remark this answers is not in the document]
Comment (Arleee1) [reply misc-bench/cpp-copy/baselines/GPU/copy.cu]: I have removed it, I apologize.

File: misc-bench/cpp-copy/baselines/CPU/copy.cpp (hunk 1 above)
Comment (fasiddique) [reply misc-bench/cpp-copy/baselines/CPU/copy.cpp:19]: This is fine, thank you!

File: misc-bench/cpp-copy/baselines/GPU/copy.cu [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+int main(int argc, char **argv)
+{
+  struct Params params = input_params(argc, argv);
+
+  uint64_t vectorSize = params.vectorLength;
+
+  std::vector<int> X;
+  std::vector<int> Y;
+
+  getVector(vectorSize, X);
```
Comment (fasiddique) [inline misc-bench/cpp-copy/baselines/GPU/copy.cu:90]: If I am not mistaken, `getVector` is a template function so you need to call it as following:

```suggestion
  getVector<int>(vectorSize, X);
```

Comment (fasiddique) [reply misc-bench/cpp-copy/baselines/GPU/copy.cu:90]: Could you please fix this in both PRs?

**Review by fasiddique — APPROVED** [review]
Thank you!

**Pull request merged**